cs4660/graph/graph.py

```diff
@@ -36,8 +36,44 @@ def construct_graph_from_file(graph, file_path):
     2. for each following line (from second line to last line), add them as edge to graph
     3. return the graph
     """
+    f = open(file_path, encoding='utf-8')
+    content = f.readlines()
+    size = (extract_nums_from_line(content[0], ' '))[0];
+    """graph = AdjacencyList()"""
+    """graph = AdjacencyMatrix()"""
+    graph = ObjectOriented()
```

Comment:
You shouldn't be manually construct any of the graph here. The graph instance is already constructed outside of the `construct_graph_from_file`.